src/proto/grpc/testing/control.proto

```diff
@@ -37,17 +37,32 @@ package grpc.testing;
 enum ClientType {
   SYNC_CLIENT = 0;
   ASYNC_CLIENT = 1;
+  CLIENT_VARIANT_1 = 2;
```

Comment:
That works well as an upper bound, do we have cases where we also want a lower-bound?  

E.g. a single-channel max bandwidth case might not be all that interesting for Android

I dont have too strong an opinion about this but worth thinking about